roles/conjur_host_identity/tasks/identity_check.yml

```diff
@@ -40,6 +40,13 @@
       conjur_validate_certs: no
   when: not ssl_configuration
 
+- block:
+  - name: conjur_validate_certs is turned off warning
+    debug: msg="Our recommendation is to use https using ssl"
+    when: item == false
+    with_items:
+        - "{{ conjur_validate_certs }}"
```

Comment:
A few notes here:
1. Using an HTTPS connection and validating certificates should have two separate warning messages

2. This message for using HTTPS connection could be moved to the block above this one, on line 36, which already is conditional on `not ssl_configuration`

   ```yaml
   - block:
     - name: Set fact "non ssl configuration"
       set_fact:
         conjur_ssl_certificate_path: ""
         conjur_validate_certs: no

     - name: Warn against using insecure connection schemes
       debug:
         msg: "[WARNING]: Provided Conjur URL uses insecure connection scheme. Please consider using HTTPS."
     when: not ssl_configuration
   ```

3. We need another warning message for when certificate validation is disabled. Seeing as it's used in the block on [line 46 of `identity.yml`](https://github.com/cyberark/ansible-conjur-collection/blob/main/roles/conjur_host_identity/tasks/identity.yml#L46), maybe we should include something there:

   ```yaml
   - block:
     - name: Warn against disabling cert validation
       debug:
         msg: "[WARNING]: Certificate validation has been disabled. Please enable with conjur_validate_certs variable."
       when: not conjur_validate_certs
         
     - name: Request identity from Conjur
       uri:
         url: "{{ conjur_appliance_url }}/host_factories/hosts"
         method: POST
         body: "id={{ conjur_host_name }}"
         headers:
           Authorization: Token token="{{ conjur_host_factory_token }}"
           Content-Type: "application/x-www-form-urlencoded"
         status_code: 201
         validate_certs: "{{ conjur_validate_certs }}"
       register: host_factory_response
       retries: 3
       delay: 10
       until: host_factory_response.status == 201
   ```

Reply:
thank you John!

Reply:
Done